client/nui/app.js

```diff
@@ -43,6 +43,28 @@ function close() {
 	}
 }
 
+function copyToClipboard(text) {
+	text = text.replace(/"/g, '');
+	text = text.trim();
+	var textarea = document.createElement('textarea');
+	textarea.value = text;
+	textarea.style.position = 'absolute';
+	textarea.style.left = '-9999px';
+	document.body.appendChild(textarea);
+	textarea.select();
+	try {
+	  document.execCommand('copy');
+	} catch (err) {
+	  console.error('Unable to copy to clipboard:', err);
+	}
+	document.body.removeChild(textarea);
+}
```

Comment:
Why make a "copy to clipboard" function? There is a clipboard API built into the browser you have replaced?

Reply:
i added this function because clipboard API not work & throws following error 
`Uncaught (in promise) NotAllowedError: Document is not focused. (@ps-camera/client/nui/index.html:0)`